test/index.ts

```diff
@@ -132,12 +132,11 @@ describe("Testing Toucan-SDK", function () {
     });
 
     it("Should automatically redeem NCT & deposit the TCO2 back", async function () {
-      // @ts-ignore
       const tco2: IToucanCarbonOffsets = new ethers.Contract(
```

Comment:
if you use `as XYZ` you don't need to explicitly declare the type here. `const tco2 = new ...` is enough